lib/hasher.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const { IdHasher, hasher: contentHasher } = require('asset-pipe-common');
+const { Readable } = require('readable-stream');
+
+module.exports = class Hasher {
```

Comment:
What's the point of a `class` if both methods are `static`?

Reply:
fair call, will switch to functions